javaclient/src/main/java/confluo/streaming/MessageListener.java

```diff
@@ -0,0 +1,9 @@
+package confluo.streaming;
+
+import confluo.rpc.Record;
+import java.util.List;
+
+public interface MessageListener {
```

Comment:
Consider replacing this with Java's [Consumer](https://docs.oracle.com/javase/8/docs/api/java/util/function/Consumer.html) class?

Reply:
done